src/Microsoft.Azure.WebJobs.Host/Queues/DispatchProcessor.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Queues
```

Comment:
>.Queues [](start = 38, length = 7)

This doesn't seem like the right namespace.   I think .Queue is for Queue  bindings; whereas the fact that this is implemented on a queue is a detail.

Reply:
changed it to "Microsoft.Azure.WebJobs.Host.Dispatch"